diff --git a/helm/kamaji/charts/kamaji-crds/templates/_helpers.tpl b/helm/kamaji/charts/kamaji-crds/templates/_helpers.tpl
index 3fe45b0..f549c07 100644
--- a/helm/kamaji/charts/kamaji-crds/templates/_helpers.tpl
+++ b/helm/kamaji/charts/kamaji-crds/templates/_helpers.tpl
@@ -46,4 +46,5 @@ app.kubernetes.io/name: {{ include "kamaji-crds.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/component: "crds"
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
 {{- end }}
