diff --git a/helm/kamaji/templates/_helpers.tpl b/helm/kamaji/templates/_helpers.tpl
index f0bf1be..28db057 100644
--- a/helm/kamaji/templates/_helpers.tpl
+++ b/helm/kamaji/templates/_helpers.tpl
@@ -40,6 +40,7 @@ helm.sh/chart: {{ include "kamaji.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+io.giantswarm.application.team: {{ index .Chart.Annotations "io.giantswarm.application.team" | quote }}
 {{- end }}
 
 {{/*
